solvers/qpSpline/PiecewisePolynomial.cpp

```diff
@@ -0,0 +1,116 @@
+#include "PiecewisePolynomial.h"
+#include <stdexcept>
+#include <cassert>
+#include <algorithm>
+
+using namespace std;
+
+PiecewisePolynomial::PiecewisePolynomial(std::vector<Polynomial> const& polynomials, std::vector<double> const& segment_times) :
```

Comment:
What does this mean concretely right now, do I have to rename or add anything? There are a lot missing methods compared to PPTrajectory of course. I can fill those in over time.

Yeah, systems/trajectories is a better place and it won't mess with Robin's stuff so I'll move it.